test/functional/data/invalid_txs.py

```diff
@@ -260,7 +259,7 @@ def get_tx(self):
         'reject_reason': "disabled opcode",
         'expect_disconnect': True,
         'get_tx': get_tx,
-        'valid_in_block' : True
+        'valid_in_block' : False
```

Comment:
nit: alternatively, could just remove the line since `False` is the default anyways, but no blocker